config/activityLog.js

```diff
@@ -124,6 +124,13 @@ module.exports = {
     lastHeartbeat: 'None',
     delta: 'None',
   },
+  missedActivity: {
+    activity: 'missedActivity',
+    generator: 'None',
+    currentCollector: 'None',
```

Comment:
I think in the activity log line we can get away with calling it just "collector" rather than "currentCollector"